packages/web/src/validators/input-validators/input-validators.ts

```diff
@@ -1,15 +1,31 @@
 import { Validator } from "../validator";
 
+let emailPattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
 export const requiredField: Validator<string> = {
-    validate: (value: string) => {
-        return value != null && value.trim() != "" ? true : false;
-    },
-    errorMessage: { "en": "Please complete the required field to continue.",  "fr": "Veuillez compléter les champs obligatoires afin de continuer." }
+  validate: (value: string) => {
+    return value != null && value.trim() != "" ? true : false;
+  },
```

Comment:
Can we make this code more concise by using implicit returns in arrow functions?
For example, this could be changed to:
`validate: (value: string) => value != null && value.trim() != "";`

The boolean expressions can also be a bit simplified to:
`value != null && value.trim() != "";`